waspc/examples/todoApp/sample.env.client

```diff
@@ -0,0 +1,2 @@
+#REACT_APP_API_URL=http://localhost:4041
+#PORT=4040
```

Comment:
I believe PORT is not used on client, so let's drop this line!

Reply:
It does actually, I checked. If PORT is set in `.env.client` file and we  run `wasp start` it will use that PORT number and open a browser page at http://localhost:4040. 

I can update it to 3000 which is default client PORT value

Reply:
Ha cool I actually forgot that :)! Awesome